tests/unit/amo/components/TestCollectionSort.js

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+
+import CollectionSort, {
+  CollectionSortBase,
+} from 'amo/components/CollectionSort';
+import { COLLECTION_SORT_NAME } from 'core/constants';
+import {
+  createFakeEvent,
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  const render = ({ ...otherProps } = {}) => {
+    const props = {
+      filters: {},
+      i18n: fakeI18n(),
+      onSortSelect: sinon.stub(),
+      ...otherProps,
+    };
+
+    return shallowUntilTarget(
+      <CollectionSort {...props} />,
+      CollectionSortBase,
+    );
+  };
+
+  it('renders a sort select', () => {
+    const sort = COLLECTION_SORT_NAME;
+    const onSortSelect = sinon.stub();
+
+    const root = render({ filters: { collectionSort: sort }, onSortSelect });
+
+    expect(root.find('.CollectionSort-label')).toHaveText('Sort add-ons by');
```

Comment:
I don't think it's useful to test statically configured data like this. It's also a maintenance burden because when you change the text you have to update the test.